registration-processor-default.properties

```diff
@@ -617,7 +617,8 @@ mosip.regproc.quality.classifier.eventbus.port=5727
 mosip.regproc.quality.classifier.server.servlet.path=/registrationprocessor/v1/qualityclassifier
 # Below quality ranges map should contain proper quality group name and quality range, any overlap of the quality 
 # range will result in a random behaviour of tagging. In range, upper and lower values are inclusive.
-mosip.regproc.quality.classifier.tagging.quality.ranges={'Poor':'0-29','Average':'30-69','Good':'70-100'}
+mosip.regproc.quality.classifier.tagging.quality.ranges={'level-1':'0-10','level-2':'10-20','level-3':'20-30','level-4':'30-40','level-5':'40-50','level-6':'50-60','level-7':'60-70','level-8':'70-80','level-9':'80-90','level-10':'90-101'}
```

Comment:
Please update the comments, is says "In range, upper and lower values are inclusive.", which is not true anymore

Reply:
Change this in other branches also, if this change is already merged

Reply:
Lets also say when score is 40, it will be level 5